packages/components/molecules/f-mega-modal/README.md

```diff
@@ -139,59 +92,20 @@ In addition to this, the modal exposes some classes which you can target in your
 | `open` | This event is emitted when the modal is opened. |
 | `close` | This event is emitted when the modal is closed. |
 
-You can add event listeners for these like so
-
-```vue
-<template>
-  <mega-modal
-    @open="onModalOpen"
-    @close="onModalClose">
-    <p>Modal content</p>
-  </mega-modal>
-</template>
-
-<script>
-export default {
-  methods: {
-    onModalOpen () {
-      // Do stuff here
-    },
-
-    onModalClose () {
-      // Do stuff here
-    }
-  }
-}
-</script>
-```
-
 ## Development
 
-Start by cloning the repository and installing the required dependencies:
```

Comment:
Think this is useful for context of what's going on below (it's straightforward to us, but may not be for someone new to JE).

Reply:
If someone didn't realise that they had to clone the repo then they wouldn't be reading this documentation anyhow.

Reply:
If that's the case, we don't need the lines below either then, as that's what the bash commands are showing.

Tbh I think this should be pulled out into storybook docs, but I think we should keep it until we replace it with those docs (or we create a JIRA explaining the docs we need to create).